src/librustdoc/clean/types.rs

```diff
@@ -100,9 +102,7 @@ impl ItemId {
     crate fn index(self) -> Option<DefIndex> {
```

Comment:
What is this used for? It seems weird.

Reply:
https://github.com/rust-lang/rust/blob/f2571a2dd68ff1d84b5df07cb38831627df6aaff/src/librustdoc/formats/cache.rs#L293

It's used here